bombardier_test.go

```diff
@@ -282,3 +284,32 @@ func TestBombardierStatsPrinting(t *testing.T) {
 // 		}
 // 	})
 // }
+
+func TestBombardierErrorIfFailToReadClientCert(t *testing.T) {
+	tlsLoadX509KeyPair = func(certFile, keyFile string) (tls.Certificate, error) {
+		return tls.Certificate{}, errors.New("failure")
+	}
+	defer func() { tlsLoadX509KeyPair = tls.LoadX509KeyPair }()
```

Comment:
You don't really need to mock this function. It will return error, since there are no "certPath"/"keyPath" files.
There is also another issue with this style of mocking. If you concurrently run multiple tests which swap this global _var_, guess what will happen? :bomb:
Get rid of this global _var_ and test actual functions.